Mailer/TwigSwiftMailer.php

```diff
@@ -41,13 +42,8 @@ class TwigSwiftMailer implements MailerInterface
 
     /**
      * TwigSwiftMailer constructor.
-     *
-     * @param \Swift_Mailer         $mailer
-     * @param UrlGeneratorInterface $router
-     * @param \Twig_Environment     $twig
-     * @param array                 $parameters
      */
-    public function __construct(\Swift_Mailer $mailer, UrlGeneratorInterface $router, \Twig_Environment $twig, array $parameters)
+    public function __construct(\Swift_Mailer $mailer, UrlGeneratorInterface $router, Environment $twig, array $parameters)
```

Comment:
The switch to namespaced Twig classes should be done in the 2.x branch, as namespaced Twig classes are already available in Twig 1.x and 2.x (the exact minor version introducing them should be checked). This way, we can allow Twig 3.x in FOSUserBundle 2.x already.

Reply:
https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2959